.env

```diff
@@ -0,0 +1,3 @@
+WEB_LISTEN_PORT_BACKEND=2222
+WEB_OUTPUT_PORT_BACKEND=2222
```

Comment:
These shouldn't be the same? If we want to use `PORT`, I can just feed that into the client.

Reply:
Yes, let's make it use `WEB_LISTEN_PORT_BACKEND` in client.

No, it can be separated if you want to have port listening externally on another number. It is good if you for example:
App working locally on 2222, but you want to listen from other systems incoming packets on port 3333. Configuration for that scenario would be:
```
WEB_LISTEN_PORT_BACKEND=2222
WEB_OUTPUT_PORT_BACKEND=3333
```